src/clients/storage/BlobStorageClient.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+let request = require('request');
```

Comment:
s/let/const

[What's the difference between let and const?](http://wesbos.com/let-vs-const/)